cmd/glasskube/cmd/namespace_options.go

```diff
@@ -27,3 +34,41 @@ func (opt *NamespaceOptions) GetActualNamespace(ctx context.Context) string {
 		}
 	}
 }
+
+func DeleteNamespace(ctx context.Context, namespace string) {
```

Comment:
This is not the right place for this function. Since we would like to introduce a similar feature for the UI soon, it would make sense to keep this feature in the `pkg/uninstall` package